static/includes/clouds-list.md

```diff
@@ -495,10 +495,15 @@ import LimitedBadge from "@site/src/components/Badges/LimitedBadge";
     <td>google-europe-north1</td>
     <td>Europe, Finland: Finland</td>
   </tr>
+  <tr>
+    <td>europe</td>
+    <td>google-europe-north2</td>
+    <td>Europe, Sweden: Stockholm</td>
+  </tr>
   <tr>
     <td>europe</td>
     <td>google-europe-southwest1</td>
-    <td>Europe, Madrid: Spain</td>
+    <td>Europe, Spain: Madrid</td>
```

Comment:
Note that the code still has it upside down:

https://github.com/aiven/aiven-core/blob/5bca4451b371dab9d36a9682a19f17b8f580d9ac/aiven/deploy/clouds/google.py#L254

I won't touch it as part of this region addition, but we can follow up on that.

Reply:
@jadkik , thank you for the approval 🙌🏻 In the docs repo, we need another approval from the doc team, and then a tech writer can merge the PR. So let me keep in touch with you and track your tickets so that I can merge when we're on prod with the region 🙂